### ISSUE TITLE
Write a guide for making a PR without GitHub CLI

### ISSUE THREAD

**Issue by michaelmaitland** [llvm/llvm-project#65192]

**Comment by tru** [comment]
Yeah I am planning on making that as well. Just haven't had time. The thing is that this flow is very well documented in the github site as well, so I didn't feel it was as pressing to work on.

**Comment by tru** [comment]
I don't have time to work on this for a while, so if you want to make a draft, feel free to push a PR and add me as a reviewer.

**Comment by michaelmaitland** [comment]
I've created a first patch in https://github.com/llvm/llvm-project/pull/65393 to help get us started.
